docs/source/user_guide/users.ipynb

```diff
@@ -0,0 +1,1750 @@
+{
```

Comment:
<p>If it's for the purpose of demonstrating where to find the APIs, I think API reference is sufficient. Folder structure of cuspatial is inherently an implementation detail and is hard to track the changes if we document them here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cuspatial/pull/666/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Agreed and removed it, replaced it with a TOC</p>